dev/dev.sh

```diff
@@ -0,0 +1,126 @@
+#―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
+# Access nix-bitcoin flake packages
+
+nb() {
+    nix build --no-link --print-out-paths --print-build-logs "$@"
+}
+
+# A package defined by nix-bitcoin
+nb .#joinmarket
```

Comment:
Should we run (parts of) the new shell scripts in ci?

Reply:
In the past, I've failed to directly run extra-container on the Cirrus CI node. I'll investigate this further and maybe later add tests for the dev scripts.